src/core/Akka.Persistence/Persistence.cs

```diff
@@ -152,9 +147,10 @@ public string PersistenceId(IActorRef actor)
         /// INTERNAL API: When starting many persistent actors at the same time the journal its data store is protected 
         /// from being overloaded by limiting number of recoveries that can be in progress at the same time.
         /// </summary>
-        internal IActorRef RecoveryPermitter()
+        internal IActorRef RecoveryPermitterFor(string journalPluginId)
```

Comment:
Same design as `AdaptersFor()` to look up recovery permitter actor per plugin basis